core/src/edu/asu/c3simulator/widgets/ProductDesigns.java

```diff
@@ -0,0 +1,149 @@
+package edu.asu.c3simulator.widgets;
+
+import com.badlogic.gdx.scenes.scene2d.InputEvent;
+import com.badlogic.gdx.scenes.scene2d.Stage;
+import com.badlogic.gdx.scenes.scene2d.ui.Image;
+import com.badlogic.gdx.scenes.scene2d.ui.Label;
+import com.badlogic.gdx.scenes.scene2d.ui.ScrollPane;
+import com.badlogic.gdx.scenes.scene2d.ui.Skin;
+import com.badlogic.gdx.scenes.scene2d.ui.Table;
+import com.badlogic.gdx.scenes.scene2d.ui.TextButton;
+import com.badlogic.gdx.scenes.scene2d.ui.Window;
+import com.badlogic.gdx.scenes.scene2d.utils.ClickListener;
+
+import edu.asu.c3simulator.util.Product;
+
+/**
+ * This class creates a popup window that display all designs to the user in order to
+ * create a new product and return the product create to the table it is going to be
+ * displayed on.
+ * 
+ * @author Alyahya, Mohammed
+ */
+public class ProductDesigns
+{
+	/**
+	 * This listener checks if any of the icons is clicked and attempts appropriate
+	 * action.
```

Comment:
What is "the appropriate action?" Is it specified by some condition or state? If so, what determines "the appropriate action?"